airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/manifest.yaml

```diff
@@ -68,7 +68,7 @@ definitions:
     description: >-
       Default Base Requester for Full Refresh streams
     type: HttpRequester
-    url_base: https://www.zopim.com/api/v2/
+    url_base: "{{ config['subdomain'] }}.zendesk.com/api/v2/chat/"
```

Comment:
```suggestion
    url_base: "https://{{ config['subdomain'] }}.zendesk.com/api/v2/chat/"
```

Reply:
Regression tests were failing with

```
Invalid URL \'.zendesk.com/api/v2/chat/routing_settings/account\': No scheme supplied. Perhaps you meant https://.zendesk.com/api/v2/chat/routing_settings/account?\\""}}\n']
```

I think that's why. Regression tests probably won't be able to compare control to target either way because new config options are now required, but.